firmware/application/apps/ui_ss_viewer.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2023 Kyle Reed
+ *
+ * This file is part of PortaPack.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#include "ui_ss_viewer.hpp"
+
+using namespace portapack;
+namespace fs = std::filesystem;
+
+namespace ui {
+
+ScreenshotViewer::ScreenshotViewer(
+    NavigationView& nav,
+    const std::filesystem::path& path)
+    : nav_{nav},
+      path_{path} {
+    set_focusable(true);
+}
+
+bool ScreenshotViewer::on_key(KeyEvent) {
+    nav_.pop();
+    return true;
+}
+
+void ScreenshotViewer::paint(Painter& painter) {
+    Style style_default{
+        .font = ui::font::fixed_8x16,
+        .background = ui::Color::black(),
+        .foreground = ui::Color::white()};
+    File file{};
+
+    painter.fill_rectangle({0, 0, 240, 320}, Color::black());
```

Comment:
Maybe use a define or a const expr for 240 (SCREEN_W) and 320 (SCREEN_H) ?

Reply:
Fixed